test/artillery/report.json

```diff
@@ -0,0 +1,3702 @@
+{
```

Comment:
I believe this could be added to the gitignore.

Reply:
Done and Done. Merge conflicts are also fixed.